docs/topic_install-f5-agent.rst

```diff
@@ -1,26 +1,65 @@
 Install the F5 OpenStack Agent
 ------------------------------
 
-.. note::
+Quick Start
+```````````
 
-    - You must have both ``pip`` and ``git`` installed on your machine in order to use these commands.
-    - It may be necessary to use ``sudo``, depending on your environment.
-
-.. topic:: To install the ``f5-openstack-agent`` package from the |openstack| branch:
+.. topic:: Install the ``f5-openstack-agent`` PyPi package using ``pip``:
 
     .. code-block:: text
 
-        $ sudo pip install git+https://github.com/F5Networks/f5-openstack-agent@liberty
+        $ sudo pip install f5-openstack-agent
 
-.. topic:: To install the ``f5-openstack-agent`` release package for v |version|:
 
-    You can install specific releases by adding ``@<release_tag>`` to the end of the install command.
+Debian Package
+``````````````
 
-    For example:
+The f5-openstack-agent package can be installed using ``dpkg`` tools.
 
-    .. code-block:: text
+1. Download and install the dependencies:
+
+.. code-block:: bash
+
+    $ curl -L -O https://github.com/F5Networks/f5-common-python/releases/download/v1.1.0/python-f5–sdk_1.1.0–1_1404_all.deb
+    $ curl –L –O https://github.com/F5Networks/f5-icontrol-rest-python/releases/download/v1.0.9/python-f5-icontrol-rest_1.0.9-1_1404_all.deb
+    $ sudo dpkg –i python-f5-icontrol-rest_1.0.9-1_1404_all.deb
+    $ sudo dpkg –i python-f5–sdk_1.1.0–1_1404_all.deb
+
+2. Download and install the f5-openstack-agent:
+
+.. code-block:: bash
+
+    $ curl -L -O https://github.com/F5Networks/f5-common-python/releases/download/v8.0.6/python-f5-openstack-agent_8.0.6-1_1404_all.deb
+    $ sudo dpkg –I python-f5-openstack-agent_8.0.6-1_1404_all.deb
```

Comment:
This should be a lower case 'i'  I know that the copy I gave you had this capitalized.